### PR TITLE
Ashar branch contrib, Prevent clicking "Save Transaction" button on modal dialog when not all fields are filled done

### PR DESCRIPTION
**Pull request by AsharAmir**
https://github.com/ktenman/portfolio/issues/9 fixed

### CODE REVIEW

**Comment by ktenman** [comment]
Thank you for your merge request. Please run `npm run format` and commit correctly formatted code

File: ui/components/portfolio-transaction-component.vue [hunk 1]
```diff
@@ -287,7 +288,12 @@ const isValidTransaction = (
     (transaction.transactionType === 'BUY' || transaction.transactionType === 'SELL') &&
     typeof transaction.quantity === 'number' &&
     typeof transaction.price === 'number' &&
-    typeof transaction.transactionDate === 'string'
+    typeof transaction.transactionDate === 'string' &&
+    transaction.instrumentId !== undefined &&
+    transaction.transactionType !== undefined &&
+    transaction.quantity !== undefined &&
+    transaction.price !== undefined &&
+    transaction.transactionDate.trim() !== ''
```
Comment (ktenman) [inline ui/components/portfolio-transaction-component.vue:296]: Please fix formatting by running this command: `npm run forma`

**Review by ktenman — CHANGES_REQUESTED** [review]

File: ui/components/portfolio-transaction-component.vue (hunk 1 above)
Comment (AsharAmir) [reply ui/components/portfolio-transaction-component.vue:300]: on it

**Comment by ktenman** [comment]
Would you happen to have any plans to fix this code because tests are failing in your branch?

**Comment by AsharAmir** [comment]
im unable to run the code on my machine, sorry

**Pull request closed without merging**